code/modules/mob/living/carbon/alien/adult/caste/hunter.dm

```diff
@@ -75,7 +78,7 @@
 					blocked = TRUE
 			if(!blocked)
 				L.visible_message(span_danger("[src] pounces on [L]!"), span_userdanger("[src] pounces on you!"))
-				L.Paralyze(100)
+				L.Paralyze(5 SECONDS)
```

Comment:
remove the paralyze or this is a meaningless change

Reply:
Pounce is very hard to land and gets blocked by shields. Its one of those things I'm comfortable with still having a hardstun, and the reason I shortened it was to make it harder to keep people stunned unless you're being more attentive to them.